### DISCUSSION TITLE
Tailwind and Stencil, tailwind mix css..

### DISCUSSION THREAD

**Discussion by rafaelgfirmino** [tailwindlabs/tailwindcss#12274] [Help] [answered]
Hey, guys.
I`m new in tailwind and i have a problem but i don't know how to solve it then.

**Declaration Clas**
<img width="625" alt="Screenshot_2023-10-21_at_07 52 26" src="https://github.com/tailwindlabs/tailwindcss/assets/5332773/f02f83be-8b83-4d56-a101-8a17497a08d6">

**Creating Apply in style.css**
![Screenshot_2023-10-20_at_21 54 31](https://github.com/tailwindlabs/tailwindcss/assets/5332773/97a8c566-2179-433a-aaad-c2c07701639e)

**Output CSS**
<img width="895" alt="Screenshot_2023-10-21_at_07 56 11" src="https://github.com/tailwindlabs/tailwindcss/assets/5332773/5cd51bd7-dcc9-4c75-bb0c-6f53ba412663">

**Image output**
<img width="403" alt="Screenshot_2023-10-20_at_21 54 37" src="https://github.com/tailwindlabs/tailwindcss/assets/5332773/2a402d69-85e4-489e-b765-052f4fa7b950">

**Comment by adydetra** [comment, marked as answer]
Use HTML only or React/Vue?

Check the installation, if you are using HTML maybe you missed a [step](https://tailwindcss.com/docs/installation)

Try checking the `tailwind.config.js` section, or provide your config code here.

I provided my code example, and it works. Try this

![gambar](https://github.com/tailwindlabs/tailwindcss/assets/108399634/fa9a7a6f-d3ec-4c20-a23f-0ad7c18c726f)

`index.html`

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link href="../dist/output.css" rel="stylesheet" />
  </head>
  <body>
    <label class="ed-label">First Name</label>
    <input class="ed-input" type="text" placeholder="ex: 4242 4242 4242 4242" />
  </body>
</html>
```

`style.css`

``` css
@tailwind base;
@tailwind components;
@tailwind utilities;

.ed-label {
  @apply text-xs font-bold text-gray-600;
}

.ed-input {
  @apply border border-gray-300 rounded-md shadow-md;
}
```

`tailwind.config.js`

```js
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
};

```

---

Run on terminal

```
npx tailwindcss -i ./src/style.css -o ./dist/output.css --watch
```

This command will generate a `dist` folder and contain the `output.css` file

---

Preview

![gambar](https://github.com/tailwindlabs/tailwindcss/assets/108399634/bcfe2ede-c7ce-4e11-9c26-1e4a587f15fa)
